src/vcr_clj/cassettes/serialization.clj

```diff
@@ -72,11 +72,11 @@
 
 (defn read-input-stream
   [hex-str]
-  (-> hex-str
-      .getBytes
-      b64/decode
-      java.io.ByteArrayInputStream.
-      serializablize-input-stream))
+  (cond-> hex-str
+    true                   .getBytes
+    (pos? (count hex-str)) b64/decode
+    true                   java.io.ByteArrayInputStream.
+    true                   serializablize-input-stream))
```

Comment:
I'd like this better keeping the top-level `->`, and just converting `b64/decode` to `(cond-> (seq hex-str) b64/decode))`